test/remote/gateways/remote_merchant_warrior_test.rb

```diff
@@ -10,7 +10,7 @@ def setup
     @credit_card = credit_card(
       '5123456789012346',
       :month => 5,
-      :year => 17,
+      :year => 18,
```

Comment:
Did you mean to include this in this PR?

Reply:
Whoops, no, ty.

Reply:
fixed